docs/en/about-us/adopters.md

```diff
@@ -158,6 +157,7 @@ The following list of companies using ClickHouse and their success stories is as
 | [Ecwid](https://www.ecwid.com/) | E-commerce SaaS | Metrics, Logging | — | — | [Slides in Russian, April 2019](https://nastachku.ru/var/files/1/presentation/backend/2_Backend_6.pdf) |
 | [Effodio](https://www.effodio.com/) | Observability, Root cause analysis | Event storage | - | - | [Blog, 2024](https://peng.fyi/post/factorial-growth-of-clickhouse-with-clause/)|
 | [Egg](https://github.com/ducc/egg) | Error Aggregation | Main Product | — | — | [GitHub repository](https://github.com/ducc/egg) |
+| [Electrum](https://www.electrum.id/) | Technology | Real-time Analytics | — | — | [Meetup Blog, October 1] |
```

Comment:
Do we have a link @chetuquib ?

Reply:
Please have it as a placeholder for now, will have the link to the meetup video this week. Thank you!

Reply:
Then we'll merge when you add

Reply:
@chetuquib do you have video?